### PR TITLE
Added log level change step in SIGUSR1 singal handler of dbg pkg.

### PR DESCRIPTION
**Pull request by siva-portworx**
**What type of PR is this?**
feature
**What this PR does / why we need it**:
Added log level change step in SIGUSR1 singal handler of dbg pkg. With this changes, we can dynamically change the log level to debug or trace and collect the collect the logs from the customer setup for debugging.

**Does this PR change a user-facing CRD or CLI?**:
No

**Is a release note needed?**:
No

**Does this change need to be cherry-picked to a release branch?**:
2.5 

Testing:
Unit tested the changes with the px-backup

### CODE REVIEW

File: pkg/dbg/dbg.go [hunk 1]
```diff
@@ -56,6 +59,30 @@ func generateFilename(name string, dir string, suffix string) string {
 	return path.Join(dir, name+"."+time.Now().Format(fnameFmt)+suffix)
 }
 
+func changeLogLevel() {
+	var level string
+	data, err := ioutil.ReadFile(logLevelFile)
+	if err != nil {
+		level = "info"
+	} else {
+		level = string(data)
```
Comment (disrani-px) [inline pkg/dbg/dbg.go:68]: Convert to lower case in case INFO or Info is given in the file

File: pkg/dbg/dbg.go [hunk 2]
```diff
@@ -17,6 +18,7 @@ import (
 const (
 	fnameFmt       = "2006-01-02T15:04:05.999999-0700MST"
 	stackTraceSize = 5120 * 1024 // 5MB
+	logLevelFile   = "/tmp/level"
```
Comment (disrani-px) [inline pkg/dbg/dbg.go:21]:
```suggestion
	logLevelFile   = "/tmp/loglevel"
```

**Review by disrani-px — APPROVED** [review]

File: pkg/dbg/dbg.go (hunk 1 above)
Comment (siva-portworx) [reply pkg/dbg/dbg.go]: Fixed.

File: pkg/dbg/dbg.go (hunk 2 above)
Comment (siva-portworx) [reply pkg/dbg/dbg.go]: Fixed.

**Pull request merged**